contracts/src/v0.8/vendor/@eth-optimism/contracts/v0.8.6/contracts/L2/predeploys/OVM_GasPriceOracle.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.6;
+pragma solidity ^0.8.6;
```

Comment:
Are vendor file changes safe? Is this copy pasted or will it get overwritten when the package is re-installed. If we can make this change then we might as well change this single line and be done with it. Don't need to copy paste out another v0.8.6 ChainSpecificUtil.